posthog/async_migrations/test/test_0002_events_sample_by.py

```diff
@@ -18,6 +19,20 @@ def execute_query(query: str) -> Any:
     return sync_execute(query)
 
 
+def mocked_execute_op_clickhouse(sql: str, query_id: str, timeout_seconds: Optional[int] = None, settings=None):
+    if sql.startswith("OPTIMIZE TABLE"):
+        raise Exception()
+
+    from ee.clickhouse.client import sync_execute
```

Comment:
Note: Rather than copying code there probably is a way to import the unpatched method if done at the top level. https://stackoverflow.com/questions/46707483/how-to-mock-a-function-that-is-imported-within-an-imported-method-from-differen